PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/Utils/PlayerSelectionOption.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PlayerSelectionOption.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import Foundation
+
+class PlayerSelectionOption : NSObject {
```

Comment:
Why do we need it to be inherited from `NSObject`?
Since it just a helper class it's may be done better
```suggestion
struct PlayerSelectionOption { ...
```
And then you'll have an init for free

Reply:
Agree :)